ifrs17/Utils/EqualityComparers.ipynb

```diff
@@ -183,6 +183,52 @@
             "metadata": {},
             "execution_count": 0,
             "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public class IdentityPropertyReader<T> where T:BaseVariableIdentity {",
```

Comment:
Please move this class in a separate notebook - I would say a new one named IdentityPropertyReader - since it doesn't fit this notebook EqualityComparers

Reply:
this is done